.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+<!-- Read comments, before committing pull request read checklist again
+
+# Checklist:
+
+- I have performed a self-review of my own code,
+- I have commented my code, particularly in hard-to-understand areas,
+- Title include "WIP" if work is in progress.
+
+-->
+
+Resolves #01 <!-- issue number -->
+
+### Description
+
+<!--
+
+- Please include a summary of the change and which issue is fixed.
+- List any dependencies that are required for this change. (gems, js libraries, etc.)
+
+-->
+
+### Type of change
+
+<!-- Please delete options that are not relevant. -->
+
+* Bug fix (non-breaking change which fixes an issue)
+* New feature (non-breaking change which adds functionality)
+* Breaking change (fix or feature that would cause existing functionality to not work as expected)
+* This change requires a documentation update
+* Documentation update
+
+### How Has This Been Tested?
+
+<!--
+
+- Please provide instructions so we can reproduce.
+- Do we need to do anything else to verify your changes?
```

Comment:
Please mention detailed steps on how to test this PR.

Reply:
Are you suggesting that the message should be:

> Please mention detailed steps on how to test this PR.

Instead of:
> Please provide instructions so we can reproduce.
> Do we need to do anything else to verify your changes?

Is that correct? @rt4914 Sir

Reply:
Something like this:
```
<!-- Steps to reproduce the behavior: -->

<!-- 1. Go to '...' -->
<!-- 2. Click on '....' -->
<!-- 3. Scroll down to '....' -->
<!-- 4. See error -->

```